synapse/lib/stormtypes.py

```diff
@@ -6379,10 +6390,15 @@ async def _methCronAdd(self, **kwargs):
                 'incvals': incval,
                 'creator': self.runt.user.iden
                 }
+
         iden = kwargs.get('iden')
         if iden:
             cdef['iden'] = iden
 
+        view = kwargs.get('view')
+        if view:
```

Comment:
Lets make this automatically select the current view if unspecified ( and always set it in the cdef )

Reply:
:+1: